frontend/packages/topology/src/components/export-app/ExportApplication.tsx

```diff
@@ -125,8 +125,13 @@ const ExportApplication: React.FC<ExportApplicationProps> = ({ namespace, isDisa
         setExportAppToast(exportAppToastConfig);
         await createExportCR();
       }
-    } catch {
-      await createExportCR();
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.error('Failed to create ExportCR. Automatically retry it once.', error);
+      createExportCR().catch((retryError) =>
+        // eslint-disable-next-line no-console
+        console.warn('Could not create ExportCR also after retring it:', retryError),
+      );
```

Comment:
@rottencandy Can you please check this code change? The `createExportCR` was introduced in #9864 to fix an issue if the Export CR is not found. I kept this as similar as possible to your solution for the moment.

Reply:
Changes look good, but I feel the first `console.error` is not necessary. The try block just checks if an ExportCR already exists, so the catch block will always execute when an export is started for the first time.
I think it would be fine to just have the second log. What do you think?

Reply:
Also, looks like `createExportCR` is called in 2 other places too. Are we catching the error only in this one place?

Reply:
The other cases are surrounded by a try-catch block in the modal component. This `createExportCR` `exportAppClickHandle` was the only one that was called without any catch.

I removed the first logline and I now see that `createExportCR` can never throw an error. But I keep the second warning now anyway.

Reply:
Looks good. Thanks!